docker.go

```diff
@@ -204,6 +297,17 @@ func commandPull(repo string) *exec.Cmd {
 	return exec.Command(dockerExe, "pull", repo)
 }
 
+func commandInspect(build Build, tag string) *exec.Cmd {
+	target := fmt.Sprintf("%s:%s", build.Name, tag)
+	args := []string{
+		"docker inspect",
+	}
+	args = append(args, target)
+	args = append(args, "> /tmp/output.json")
+
+	return exec.Command(bashExe, "-c", strings.Join(args, " "))
```

Comment:
bash won't be present in windows image. Also, /tmp directory would be present in windows.

Reply:
Use this command instead:
`
exec.Command("docker", "inspect")
`
and redirect stdout and stderr to /tmp/output.json file

Reply:
@shubham149 - I missed your updates where you've added in windows support, which is the reason this was missed. Will look into it

Reply:
instead of using bash and writing to a file, you can read directly from `exec.Cmd.Stdout`

Reply:
yeah this has been cleaned up.